test/meta-colony.js

```diff
@@ -389,8 +389,15 @@ contract("Meta Colony", accounts => {
 
     it("should be able to set domain on task", async () => {
       await colony.addDomain(3);
-      await colony.makeTask(SPECIFICATION_HASH, 1);
-      await colony.setTaskDomain(1, 2);
+      const { logs } = await colony.makeTask(SPECIFICATION_HASH, 1);
+      const taskId = logs[0].args.id.toNumber();
+      await colony.setTaskRoleUser(1, WORKER_ROLE, WORKER);
```

Comment:
Now that we can execute with just 1 signature, let's keep test data setup simpler. You can remove this line and execute just with the Manager's signature.